gulpfile.babel.js

```diff
@@ -213,116 +213,11 @@ const bundleScripts = watch => {
 
   return gulp.src(['./assets/js/polyfills.js', './assets/js/components.js'])
     .pipe(bundler())
-    .pipe(gulp.dest('./public/assets/scripts'));
-
-
-  /*return browserify(['./assets/js/polyfills.js', './assets/js/components.js'])
```

Comment:
If we don't need this code anymore then we should delete it.

Reply:
@saedwards can you resolve this one then I can get this into the v1.4.2 release. Thanks.